App/Components/RoundedCornersView.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import { View, ViewProps, LayoutChangeEvent, ViewStyle, StyleProp } from 'react-native'
+
+interface State {
+  borderRadius: number
+}
+
+interface Props extends ViewProps {
+  radiusToHeightRatio: number
+}
+
+class RoundedCornersView extends Component<Props, State> {
```

Comment:
Use this if you ever need to create a view that automatically has nice rounded corners. This defaults to radius to height ratio of 0.2. If we use it, our corner radiuses all over the will be consistent.